pytorch_lightning/loops/dataloader/evaluation_loop.py

```diff
@@ -266,3 +269,9 @@ def on_evaluation_epoch_end(self) -> None:
         self.trainer.call_hook(hook_name)
         self.trainer.call_hook("on_epoch_end")
         self.trainer.logger_connector.on_epoch_end()
+
+    def state_dict(self) -> Dict:
+        return {}
+
+    def load_state_dict(self, state_dict: Dict) -> None:
+        raise NotImplementedError
```

Comment:
Why does this need to raise an Error? If there's is nothing to `load` then it can just `pass`

Reply:
Done !